services/QuillLMS/app/models/user.rb

```diff
@@ -668,6 +668,14 @@ def set_time_zone
     end
   end
 
+  def duplicate_empty_student_accounts
+    User
+      .student
+      .where(email: email)
+      .where.not(id: id)
+      .where.missing(:activity_sessions, :students_classrooms)
```

Comment:
This syntax is new to me and handy!

Reply:
Just got added in Rails 6.1

Reply:
Not exactly an objection, but is this something we want to put on the `User` model when we're aiming to clean up all instances of this in this very PR?  Like, this should always be, effectively, a null operation after the rake task runs?

Reply:
Good question.  Long term, I think this method should be removed, However, the rake task will clean up some, but not all of the duplicates and I see this method being useful until the duplicates are removed.  For example, consider user1 and user2 with the same email.  [Merging](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/concerns/student.rb#L121) user2 into user1 means that user1.duplicate_empty_student_accounts returns user2.  In which case I can re-run the rake task.

Reply:
Oh, fun, I didn't know about the `.missing` method, neat.

Very Oliver Twist.
![Screen Shot 2022-10-31 at 10 51 37 AM](https://user-images.githubusercontent.com/1304933/199037375-dc82468a-9275-477c-8a1b-d8672c56e6de.png)